src/components/Receiving/ItemDetailsFooter.js

```diff
@@ -11,8 +11,10 @@ const ItemDetailsFooter = ({
   checkedItemsListLength,
   close,
   currentLine,
+  isReceiveButtonDisabled,
   onClickNext,
   onClickPrevious,
+  onClickReceive,
```

Comment:
this is a very tight coupling, I think it's better to stay more generic. You have a prop `onClickNext` already, you could reuse it.

Reply:
Updated